bench-tps/src/cli.rs

```diff
@@ -396,6 +399,13 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .validator(is_keypair)
                 .help("File containing the node identity (keypair) of a validator with active stake. This allows communicating with network using staked connection"),
         )
+        .arg(
+            Arg::with_name("commitment_config")
+                .long("commitment-config")
+                .takes_value(true)
+                .possible_values(&["processed", "confirmed", "finalized"])
+                .help("Block commitment config for getting latest blockhash. default: processed"),
```

Comment:
```suggestion
                .possible_values(&["processed", "confirmed", "finalized"])
                .default_value("processed")
                .help("Block commitment config for getting latest blockhash"),
```
You can specify the default value up here; in doing so, I think the default will get displayed in the help message so you can remove the explicit test (build + run `--help` to confirm)

Reply:
I don't have strong opinions on what the default is, but the point stands that the default value (whatever it may be) can be specified here to simplify / obviate the subsequent arg parsing

And fwiw, there will seemingly be some change in value for commitment level somewhere:
- RpcClient is currently created with `confirmed`
- ThinClient is currently created with `finalized`
    - Via this which eventually gets called:
    https://github.com/solana-labs/solana/blob/3284d34e041f737ddb0b946d0695d9c6ada0fc71/rpc-client/src/rpc_client.rs#L254
- TpuClient is currently created with `confirmed`

Maybe since we had 2/3 with `confirmed`, we do go that route ?

Reply:
good point on the title. and agree on the use of `confirmed` as the default